src/core/processors/compositeprocessorutils.cpp

```diff
@@ -283,7 +285,7 @@ void util::expandCompositeProcessorIntoNetwork(CompositeProcessor& composite) {
             network.addLink(l);
         }
 
-        network.removeProcessor(&composite);
+        auto tmp = network.removeProcessor(&composite);
```

Comment:
Does tmp have something to do with deleting the composite?
Either name it accordingly or add a comment in that case.

Reply:
don't think is was needed with the holder up top